src/openrct2/park/ParkFile.cpp

```diff
@@ -422,7 +422,20 @@ namespace OpenRCT2
         void ReadWriteGeneralChunk(OrcaStream& os)
         {
             auto found = os.ReadWriteChunk(ParkFileChunkType::GENERAL, [this](OrcaStream::ChunkStream& cs) {
-                cs.ReadWrite(gGamePaused);
+                if (cs.GetMode() == OrcaStream::Mode::READING)
+                {
+                    cs.ReadWrite(gGamePaused);
+                    // The flags have been stored in some park files, remove them
+                    // as they are used to temporarily pause the game, only user driven state.
+                    gGamePaused &= ~GAME_PAUSED_MODAL;
+                    gGamePaused &= ~GAME_PAUSED_SAVING_TRACK;
```

Comment:
Wouldn't you want to do something like this instead?
```cpp
auto isPaused = cs.Read<uint8_t>();
gGamePaused &= ~GAME_PAUSED_NORMAL;
gGamePaused |= (isPaused & GAME_PAUSED_NORMAL);
```

Reply:
Well I think I understand what you want but then wouldn't it be simpler to simply test the single bit and just assign it like:
```cpp
gGamePaused = (cs.Read<uint8_t>() & GAME_PAUSED_NORMAL);
```
Just masks of the single state and directly assigns it.

Reply:
I wanted to be explicit about why we deal with the uint8_t the way it does since we use only a single bit of it, the logical way would be to use a bool then but if we would simply ignore the extra bits then its UB (always evalutes to true due to the codegen), but I won't mind simplifying it since we also didn't officially release NSF yet.

Reply:
Since we only care about the user pause, &-ing with `~GAME_PAUSED_NORMAL` makes more sense than removing the other flags. It doesn't really matter how exactly.

```cpp
cs.ReadWrite(gGamePaused);
gGamePaused &= GAME_PAUSED_NORMAL;
```

Reply:
@ZehMatt I did it that way to preserve the other flags. I don't know if that your way would have consequences or not.

Reply:
The other flags should not be imported, they have been exported by accident.

Reply:
I went with @IntelOrca suggestion and only import/export the state of GAME_PAUSED_NORMAL in gGamePaused